### ISSUE TITLE
Unable to run Sourcetrail over Xming

### ISSUE THREAD

**Issue by ruipires** [CoatiSoftware/SourcetrailBugTracker#667]
I'm trying to run Sourcetrail in WSL, using Xming for X11 forwarding.

I am able to run other X11 applications (namely xterm & Clion).

WSL is set up to use Ubuntu ( Ubuntu 18.04.1 LTS (GNU/Linux 4.4.0-17134-Microsoft x86_64) ).

` ~/opt/Sourcetrail/Sourcetrail.sh -v
Sourcetrail Version 2018.4.45
`

Trying to start up will result in:
` ~/opt/Sourcetrail/Sourcetrail.sh
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-rui'
failed to get the current screen resources
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig error: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 72: non-double matrix element
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 80: saw unknown, expected number
The X11 connection broke: I/O error (code 1)
XIO:  fatal IO error 2 (No such file or directory) on X server "localhost:0.0"
      after 358 requests (353 known processed) with 0 events remaining.
terminate called without an active exception
fish: '~/opt/Sourcetrail/Sourcetrail.sh' terminated by signal SIGABRT (Abort)` 

Unable to extract more logs, since UI is unusable.

**Comment by egraether** [comment]
Thanks for reporting! I couldn't find any solution online so far. We need to look into this...

**Comment by ruipires** [comment]
I could figure out what was going on.
Tried installing the XMing extra fonts package, but that didn't help also.

**Comment by egraether** [comment]
No, it's not a fonts problem, those errors are unrelated. That's the important part:
```line 80: saw unknown, expected number The X11 connection broke: I/O error (code 1) XIO: fatal IO error 2 (No such file or directory) on X server "localhost:0.0" after 358 requests (353 known processed) with 0 events remaining. terminate called without an active exception fish: '~/opt/Sourcetrail/Sourcetrail.sh' terminated by signal SIGABRT (Abort)```

**Comment by ruipires** [comment]
Found a related issue for another project: https://github.com/ros-visualization/rqt_image_view/issues/9

The solution/workaround is also applicable here: VcXsrv (instead of XMing) works.
